app/views/course/events/_event.html.slim

```diff
@@ -0,0 +1,38 @@
+- if can?(:read, event)
+  = div_for(event)
+
+    a role="button" data-toggle="collapse" href="#event#{event.id}-body"
+      li.list-group-item
+
+        .pull-right
+          = format_datetime(event.start_time)
+          - if event.end_time.present?
+            = t('.to')
```

Comment:
This seems potentially nasty.what if the language is not a left-to-right language?